frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -214,6 +214,11 @@ function ReaderHighlight:init()
         }
     end)
 
+    -- default value for non-touch content selection speed-up rate
+    if G_reader_settings:hasNot("highlight_non_touch_spedup") then
+        G_reader_settings:makeTrue("highlight_non_touch_spedup")
```

Comment:
`nilOrTrue` may help to avoid initializing this.
In any case it should not exist in the touch device settings.

Reply:
May or will?

Reply:
https://github.com/koreader/koreader/blob/d22af656fbb1303ac7d45715292e56d0f2076c7d/frontend/apps/reader/modules/readerhighlight.lua#L2243
`nilOrTrue`

Reply:
https://github.com/koreader/koreader/blob/d22af656fbb1303ac7d45715292e56d0f2076c7d/frontend/apps/reader/modules/readerhighlight.lua#L643
`flip...`